tests/framework/clients/dynamic/dynamic.go

```diff
@@ -67,3 +68,20 @@ func (c *ResourceClient) Create(ctx context.Context, obj *unstructured.Unstructu
 
 	return c.ResourceInterface.Create(ctx, obj, opts, subresources...)
 }
+
+// GetRancherDynamicClient is a helper function that instantiates a dynamic client to communicate with the rancher host.
+func GetRancherDynamicClient(bearerToken string, rancherConfig *rancher.Config, session *session.Session) (dynamic.Interface, error) {
```

Comment:
can we make this a method of the rancher client instead?

Reply:
Do you mean move this to the rancher client?

Reply:
Ah I see what you mean